app/models/miq_server/environment_management.rb

```diff
@@ -27,9 +27,6 @@ def minimal_env_options
       @minimal_env_options ||= begin
         options = self.minimal_env? ? spartan_mode.split(MIQ_SPARTAN_ROLE_SEPARATOR) : []
         options.shift # remove the "minimal" from the front of the array
```

Comment:
Since you removed the line below, won't this line now be the last line...i.e. won't this cause the `@minimal_env_options` to only be the shifted item?